### PR TITLE
enqueue/simple-client dependency resolution update

### PR DESCRIPTION
**Pull request by mikeweb85**
resolving issues related to symphony/config and enquque/fs

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -19,15 +19,15 @@
     },
     "require": {
         "php": ">=7.2.0",
-        "cakephp/cakephp": "^4.0",
-        "enqueue/simple-client": "^0.9",
+        "cakephp/cakephp": "~4.0",
```
Comment (othercorey) [inline composer.json:22]: What is this for?

Comment (mikeweb85) [reply composer.json:22]: Installation with CakePHP 4.0 and 4.1 generates the following error
---
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Conclusion: don't install enqueue/simple-client 0.10.1 (conflict analysis result)
    - Conclusion: don't install enqueue/fs 0.9.12 (conflict analysis result)
    - Conclusion: don't install enqueue/simple-client 0.10.4 (conflict analysis result)
    - enqueue/fs 0.9.0 requires enqueue/dsn 0.9.x-dev -> found enqueue/dsn[dev-master, 0.9.0, ..., 0.10.x-dev (alias of dev-master)] but it does not match the constraint.
    - Conclusion: don't install enqueue/fs 0.9.8 (conflict analysis result)
    - Conclusion: don't install enqueue/enqueue 0.10.6 (conflict analysis result)
    - enqueue/fs 0.9.1 requires queue-interop/queue-interop ^0.7 -> satisfiable by queue-interop/queue-interop[0.7.0, 0.7.1].
    - You can only install one version of a package, so only one of these can be installed: queue-interop/queue-interop[0.7.0, 0.7.1, 0.8.0].
    - enqueue/simple-client 0.10.0 requires queue-interop/queue-interop ^0.8 -> satisfiable by queue-interop/queue-interop[0.8.0].
    - Root composer.json requires enqueue/simple-client ~0.10 -> satisfiable by enqueue/simple-client[0.10.0, 0.10.1, 0.10.4].
    - enqueue/simple-client 0.10.0 requires enqueue/enqueue ^0.10 -> satisfiable by enqueue/enqueue[0.10.0, ..., 0.10.6].
    - Conclusion: don't install one of enqueue/enqueue[0.10.4], enqueue/fs[0.9.2] (conflict analysis result)
    - Conclusion: don't install one of enqueue/enqueue[0.10.3], enqueue/fs[0.9.2] (conflict analysis result)
    - Conclusion: don't install one of enqueue/enqueue[0.10.1], enqueue/fs[0.9.2] (conflict analysis result)
    - Conclusion: don't install one of enqueue/dsn[0.10.4], enqueue/fs[0.9.2] (conflict analysis result)
    - Root composer.json requires enqueue/fs ^0.9 -> satisfiable by enqueue/fs[0.9.0, ..., 0.9.12].
    - enqueue/enqueue 0.10.0 requires enqueue/dsn ^0.10 -> satisfiable by enqueue/dsn[0.10.0, 0.10.1, 0.10.4].
    - You can only install one version of a package, so only one of these can be installed: enqueue/dsn[0.9.0, ..., 0.10.4].
    - enqueue/fs 0.9.2 requires enqueue/dsn ^0.9 -> satisfiable by enqueue/dsn[0.9.0, 0.9.2].
    - Conclusion: don't install enqueue/dsn[0.9.2] | install one of enqueue/enqueue[0.10.4], enqueue/dsn[0.10.4] (conflict analysis result)

Comment (mikeweb85) [reply composer.json:22]: updated symfony/config included with the CakePHP/app, updated to match essentially

**Comment by markstory** [comment]
Thank you 👍

**Pull request merged**